io.qt.qtwebengine.BaseApp.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id" : "io.qt.qtwebengine.BaseApp",
+    "branch": "5.13",
+    "runtime" : "org.kde.Platform",
+    "sdk" : "org.kde.Sdk",
+    "runtime-version" : "5.13",
+    "modules" : [
+        {
+            "name" : "qt5-qtwebengine",
+            "buildsystem" : "qmake",
+            "cleanup-platform" : [
+                "/bin",
+                "/mkspecs"
+            ],
+            "make-install-args": ["INSTALL_ROOT=/app/potatoes"],
+            "config-opts" : [
+                "--",
+                "-system-ffmpeg",
+                "-webengine-proprietary-codecs",
+                "-webengine-icu",
+                "-webp"
+            ],
+            "post-install" : [
+                "cp -r /app/potatoes/usr/* /app",
+                "rm -rf /app/potatoes"
+            ],
+            "modules" : [
+                "shared-modules/python2.7/python-2.7.json",
+                {
+                    "name" : "pciutils",
+                    "no-autogen" : true,
+                    "make-args" : [
+                        "SHARED=no",
+                        "ZLIB=no",
+                        "PREFIX=/app"
+                    ],
+                    "post-install": [
+                        "make install-pcilib",
+                        "make install-lib"
+                    ],
+                    "sources" : [
+                        {
+                            "type" : "git",
+                            "url" : "https://kernel.googlesource.com/pub/scm/utils/pciutils/pciutils",
+                            "branch" : "v3.6.2"
+                        },
+                        {
+                            "type" : "shell",
+                            "commands" : [
+                                "sed -i 's@PREFIX=/usr/local@PREFIX=/app@' Makefile"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name" : "opus",
+                    "sources" : [
+                        {
+                            "type" : "git",
+                            "url" : "https://github.com/xiph/opus.git",
+                            "branch" : "v1.3.1"
+                        }
+                    ]
+                }
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://download.qt.io/archive/qt/5.13/5.13.2/submodules/qtwebengine-everywhere-src-5.13.2.tar.xz",
+                    "sha256" : "adcf56b5de6f34744bba2307b696fc75611884f4270e87dfa760d6e99dd711bb"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/resources.patch"
+                }
+            ]
+        },
+        {
+            "name": "polish",
```

Comment:
I would prefer to have it done in `post-install`.

Reply:
Why? I did it this way so it's easier to modify these hacks without having to rebuild all of qtwebengine every time.

Reply:
I guess the question is if you really want to install into dir called `potatoes` and why.

Reply:
So be it, I guess >40 minutes of build can justify that.